vizro-ai/tests/integration/test_dashboard.py

```diff
@@ -55,97 +203,23 @@ def test_simple_dashboard(dash_duo, model_name):  # noqa: PLR0915
     Add a filter to filter the scatter plot by continent.
     Add a second filter to filter the chart by year.
     """
-    report_dir = "tests/integration/reports"
-    os.makedirs(report_dir, exist_ok=True)
-
-    try:
-        vizro_type = os.environ["VIZRO_TYPE"]
-    except KeyError:
-        vizro_type = "local_env"
 
     dashboard = vizro_ai.dashboard([df1, df2], input_text)
     app = Vizro().build(dashboard).dash
     dash_duo.start_server(app)
     assert dash_duo.get_logs() == []
```

Comment:
Does this get added to the score? I think this is the most important metric :)

Reply:
Are you talking about the browser logs or start  of the app? Neither of it is included to score. If this step will fail it will not count anything.

Reply:
I mean the fact that the dashboard is even starting - that should be definitely in the score, as this is after all the most important metric